pandas/tests/io/excel/test_readers.py

```diff
@@ -1278,6 +1278,13 @@ def test_ignore_chartsheets_by_int(self, request, read_ext):
         ):
             pd.read_excel("chartsheet" + read_ext, sheet_name=1)
 
+    def test_dtype_dict(self, read_ext):
```

Comment:
Can you please leave a link to the relevant github issue here? And maybe also make the name of this test more specific, eg in this care we about about `dtype` argument not being modified in the case of duplicate columns being present